### PR TITLE
Changelog for 11-29-2024

### PR DESCRIPTION
**Pull request by bgrenon**
https://neon-next-git-docs-changelog-11-29-2024-neondatabase.vercel.app/docs/changelog

### CODE REVIEW

File: content/changelog/2024-11-29.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+![get started with your new org](/docs/manage/org_projects.png)
+
+Create a new organization, transfer over your projects, invite your team members and get started collaborating. The Organizations feature lets you manage all of your team's projects under a single account — with billing, role management, and project transfer capabilities in one accessible location.
+
+See [Neon Organizations](/docs/manage/organizations) to get started. Organizations is a paid account feature.
+
+## Organization API keys
+
+Along with GA for Organizations, we're also announcing support for [Organizations](https://neon.tech/docs/manage/organizations). As an admin of an organization, you control your organization's API keys. These keys provide admin-level access to all organization resources, including projects, members, and billing information. See [Organization API keys](/docs/manage/orgs-api).
```
Comment (clarkbw) [inline content/changelog/2024-11-29.md:17]: >Along with GA for Organizations, we're also announcing support for [Organizations](https://neon.tech/docs/manage/organizations).

i don't understand this sentence.

Comment (bgrenon) [reply content/changelog/2024-11-29.md:17]: Ah, that should have been "Organization API Keys"

Comment (bgrenon) [reply content/changelog/2024-11-29.md]: changed to:
Along with GA for Organizations, we're also announcing support for [Organization API Keys](/docs/manage/orgs-api). As an admin of an organization, you control your organization's API keys. These keys provide admin-level access to all organization resources, including projects, members, and billing information.

File: content/changelog/2024-11-29.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
+---
+description: Organizations are GA! Also, a new Schema Diff GitHub Action, Postgres version updates, and more.
+---
+
+### Neon Organizations are GA! 🎉
+
+We're happy to announce that Neon **Organizations** is now **generally available** for all everyone looking to move their teams to Neon.
```
Comment (pffigueiredo) [inline content/changelog/2024-11-29.md:7]: nit: "for everyone"?

File: content/changelog/2024-11-29.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+  - We've modified our autoscaling range slider to better handle larger compute sizes and added a constraint: your maximum range can be no more than 8x your minimum.
+
+<div style={{ display: 'flex', justifyContent: 'center'}}>
+  <img src="/docs/relnotes/autoscaling_ratio.png" alt="autoscaling slider showing ratio limit of 8x min" style={{ width: '60%', maxWidth: '600px', height: 'auto' }} />
+</div>
+
+- **Fixes**
+
+  - Improved validation of API key names: added length limits to the API specification and improved whitespace handling in the UI.
```
Comment (danieltprice) [inline content/changelog/2024-11-29.md:102]:
```suggestion
  - Improved validation of API key names: added a 64-character length limit for API key names to the API specification and improved whitespace handling in the UI.
```

File: content/changelog/2024-11-29.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+  - We've added a **Monitor** button to each listed endpoint on your Branch details page. Click the button to open the **Monitoring** page, displaying metrics for this endpoint.
+
+    ![monitor button on endpoint item](/docs/relnotes/monitor_button_endpoint_item.png)
+
+  - Added quick compute size editing directly from the Branches page — just click the size link in the primary branch column to adjust your settings.
+
+    ![branches table compute drawer](/docs/relnotes/branches_table_compute_drawer.png)
+
+  - We've modified our autoscaling range slider to better handle larger compute sizes and added a constraint: your maximum range can be no more than 8x your minimum.
```
Comment (danieltprice) [inline content/changelog/2024-11-29.md:94]: remove this item. It's still under a feature flag, only visible to Neon employees.

Comment (bgrenon) [reply content/changelog/2024-11-29.md]: removed

**Review by danieltprice — APPROVED** [review]

**Pull request merged**